src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx

```diff
@@ -66,7 +66,10 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
     if (this.state.istioObjectDetails) {
       const objectType = dicIstioType[this.props.match.params.objectType];
       const methodName = objectType.charAt(0).toLowerCase() + objectType.slice(1);
-      title = this.state.istioObjectDetails[methodName].metadata.name;
+      const object = this.state.istioObjectDetails[methodName];
+      if (object) {
+        title = this.state.istioObjectDetails[methodName].metadata.name;
```

Comment:
it sounds more logical to use `object.metadata.name` here as you just created that variable, but it's ok, I'm nitpicking :-)

Reply:
My head! :see_no_evil: Don't judge me, hahaha, bad day today :)
I wanted to do that, but probably changed context for some reason... Thanks for the catch!

Reply:
I don't judge it's such a tiny thing :)

Reply:
I had so few things to review that it was an easy catch :)